examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -167,6 +167,12 @@ affinity: {}
 # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
 nodeSelector: {}
 
+# Kubernetes deployment labels to apply
+# https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+additionalLabels: {
```

Comment:
Also, this should probably be called `additionalDeploymentLabels` as it only applies to the `Deployment`/`StatefulSet` and not to _everything_ with labels that the chart creates. I don't like using `deployment` in the name because it also applies to the `StatefulSet`, but I can't think of a better term to capture both things while remaining clear.